### PR TITLE
add price async recipe

### PR DESCRIPTION
**Pull request by hiagolcm**
**What problem is this solving?**

We are implementing a new feature to the product-summary: the [asynchronous price](https://github.com/vtex-apps/product-summary/pull/280). This PR adds a guide to help the developer to implement this functionality.

**How should this be manually tested?**
[workspace](hiago--storecomponents.myvtex.com)

**Screenshots or example usage:**
priceasync

### CODE REVIEW

File: docs/en/Recipes/layout/async-price.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    "children": [
+      // other children
+-     "product-list-price#summary",
+-     "flex-layout.row#selling-price-savings",
+-     "product-installments#summary",
+-     "add-to-cart-button",
++     "product-price-suspense"
+   ]
+  }
+}
```
Comment (klzns) [inline docs/en/Recipes/layout/async-price.md:86]:
```suggestion
  "product-summary.shelf": {
    "props": {
     "isPriceAsync": true
    },
    "children": [
      // other children
-     "product-list-price#summary",
-     "flex-layout.row#selling-price-savings",
-     "product-installments#summary",
-     "add-to-cart-button",
+     "product-price-suspense"
   ]
  },
+ "product-price-suspense": {
+   "children": [
+     "product-list-price#summary",
+     "flex-layout.row#selling-price-savings",
+     "product-installments#summary",
+     "add-to-cart-button"
+   ]
+ },
}
```

File: docs/pt/Recipes/layout/preco-assincrono.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    },
+    "children": [
+      // other children
+-     "product-list-price#summary",
+-     "flex-layout.row#selling-price-savings",
+-     "product-installments#summary",
+-     "add-to-cart-button",
++     "product-price-suspense"
+   ]
+  }
```
Comment (klzns) [inline docs/pt/Recipes/layout/preco-assincrono.md:85]: Same as above

File: docs/pt/Recipes/layout/preco-assincrono.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+---
+title: Adicionado preço assíncrono no product-summary
+description: "Tenha o preço mais atualizado sem ter que lidar com um aumento no tempo de resposta da página"
+date: "21/10/2020"
+tags: ["price", "simulation"]
```
Comment (klzns) [inline docs/pt/Recipes/layout/preco-assincrono.md:5]: No need for this one, we only render in English

File: docs/en/Recipes/layout/async-price.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    // other children
+    "product-list-price#summary",
+    "flex-layout.row#selling-price-savings",
+    "product-installments#summary",
+    "add-to-cart-button",
+  ]
+}
+```
+
+4. Wrap the price information in the `product-price-suspense`.
```
Comment (klzns) [inline docs/en/Recipes/layout/async-price.md:61]:
```suggestion
4. Wrap the price blocks in the `product-price-suspense`.
```

**Review by klzns — CHANGES_REQUESTED** [review]

File: docs/pt/Recipes/layout/preco-assincrono.md (hunk 3 above)
Comment (hiagolcm) [reply docs/pt/Recipes/layout/preco-assincrono.md]: You mean the whole file, right?
Done 👍

**Review by klzns — APPROVED** [review]

**Pull request merged**